docs/DeveloperGuide.md

```diff
@@ -394,23 +394,48 @@ _{Explain here how the data archiving feature will be implemented}_
 
 #### Implementation
 
-This feature is mainly implemented by the `CustomiseCommand` and `PersonCard` classes. The following methods are the main ones:
+This feature is mainly implemented by the `CustomiseOrderCommand` and `PersonCard` classes. The following methods are the main ones:
 
-* `CustomiseCommand#changeAttributeOrder(Model model)` - saves the new attribute order in the `preferences.json` file.
+* `CustomiseOrderCommand#execute()` - saves the new attribute order in the `preferences.json` file by calling `Model#setGuiSettings()`.
 * `PersonCard#setAttributes()` - reads the order from the `preferences.json` file and builds the `PersonCard` in the order specified.
 
-_{diagrams and more in depth explanation to be added}_
+Given below is an example usage scenario and how this feature behaves at each part of the mechanism.
+
+Step 1. The user launches the application. Under each contacts' name, the current order the user sees is: Tags, phone number, email and address (This is also the default order). The user wants to have each email right below each name instead.
+
+Step 2. The user executes `customise order e/` to make the emails appear right below the name.
+
+* Step 2a. `SoConnectParaser` processes the input and calls the parser `CustomiseCommandParser` to parse `" order e/"`
+* Step 2b. `CustomiseCommandParser` processes the input and calls the parser `CustomiseOrderCommandParser` to parse `" e/"`.
+* Step 2c. `CustomiseOrderCommandParser` processes the input and adds to a list of attributes with `EMAIL` and then fills in the rest of the attributes in the following default order `TAGS`, `PHONE` and lastly `ADDRESS`.
```

Comment:
`adds to a list of attributes with EMAIL` This portion is unclear / hard to understand

Reply:
Ok, I shall fix it.